src/components/LogIn/index.ts

```diff
@@ -1,2 +1 @@
 export { default } from "./LogIn";
-export { Label as LogInLabel } from "./types";
```

Comment:
I added these exports in my last PR so that we can always import the label by the full name to avoid conflicts instead of having to do `import { Label as LogInLabel } ...`. Was there a reason you removed this?